lib/publiccloud/eks.pm

```diff
@@ -9,43 +9,25 @@
 # Documentation: https://docs.aws.amazon.com/AmazonECR/latest/userguide/docker-push-ecr-image.html
 
 package publiccloud::eks;
-use Mojo::Base 'publiccloud::aws';
+use Mojo::Base 'publiccloud::k8s_provider';
 use testapi;
 use utils;
 
 has security_token => undef;
```

Comment:
how this correlate with `security_token` which we already have in `aws_client` ?

Reply:
Done